### ISSUE TITLE
The device doesn't work

### ISSUE THREAD

**Issue by AdoreTu** [sylvek/itracing2#36]
Other than the fact that my phone detect that there is a device the device does not beat nor does it do anything to make my phone ring when I set it up in both of the applications for the device. The device will not be when I turn it on or off so its hard to tell if it is on or off I can see on the map that it is detecting a GPS but that is about it so it is basically useless

**Comment by sylvek** [comment]
please try the beta version and tell me if it is better
https://play.google.com/apps/testing/net.sylvek.itracing2

**Comment by AdoreTu** [comment]
Thank you for your reply unfortunately my device does not work I was able
to get it to track the device once using the GPS program on the
application. I did sign up for the beta version that you suggested.
Unfortunately it still does not find my device & I can not tell when the
device is on or off because for some reason the sound on the device is not
working, therefore it makes the device pretty much useless. I also I've
tried using the device just as to try contacting the phone and I've made
attempts to turn it on and off several times because I'm not sure it when
its on or off because so far I have not been able to get any sound from the
device. It did not make contact to my cell phone, so I guess my device is
defective. I am supposed to be receiving a new one from the seller. I am
hoping that that one will work but last time I ordered it, took almost 2
months to receive.  I'm not sure when I will have the device. It came new
with a battery. I do not have another battery of this size to test in it.
Thank you for your reply and helpful attempt. God Bless
